DeviceTests/Caboodle.DeviceTests.Shared/Traits.cs

```diff
@@ -8,6 +11,9 @@ internal static class DeviceTypes
         {
             public const string Physical = "Physical";
             public const string Virtual = "Virtual";
+
+            internal static string ToExclude =>
+                DeviceInfo.DeviceType == RuntimeDeviceType.Physical ? Virtual : Physical;
```

Comment:
Something about this wording/idea is a bit annoying to my mind, but I can't really think of a better/different way right now.

Reply:
Yeah :( I agree...